.github/workflows/registries.yml

```diff
@@ -125,3 +125,40 @@ jobs:
           OCI_GITHUB_REGISTRY_URL: ${{ secrets.OCI_GITHUB_REGISTRY_URL }}
           OCI_GITHUB_REGISTRY_USERNAME: ${{ secrets.OCI_GITHUB_REGISTRY_USERNAME }}
           OCI_GITHUB_REGISTRY_PASSWORD: ${{ secrets.OCI_GITHUB_REGISTRY_PASSWORD }}
+
+  elastic:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup
+        uses: ./.github/actions/setup_shellspec # use local action
+              
+      -
+        name: Login to Elastic
+        uses: docker/login-action@v1
```

Comment:
This is just for clarification. The step's name is `Login to Elastic`, yet the action logs in to docker. Would you mind explaining it please?

Reply:
The step means log into the Elastic registry, but I have found it confusing before too when reading it. I'll update all the steps to be a bit clearer 👍